src/js/rtc_session.js

```diff
@@ -802,6 +884,7 @@ export default class RtcSession {
         }
         if (evt.track.kind === 'video' && this._remoteVideoElement) {
             this._remoteVideoElement.srcObject = evt.streams[0];
+            this._remoteVideoStream = evt.streams[0];
```

Comment:
it's not binding with getUserAudioStats, so even if user never calls it,  we're still able to retrieve the remote stream.